moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -547,8 +547,11 @@ const ProjectFundingTable = () => {
                   ...newData,
                   project_id: projectId,
                   // preventing empty strings from being saved
-                  funding_description: newData.funding_description || null,
-                  funding_amount: newData.funding_amount || null,
```

Comment:
this wasnt actually preventing empty strings from being saved also not necessary for the funding amount